plugin/vim-gh-line.vim

```diff
@@ -46,7 +46,12 @@ func! s:gh_line() range
 
     " Git Commands
     let origin = system(cdDir . "git config --get remote.origin.url" . " | " . sed_cmd)
-    let branch = system(cdDir . "git rev-parse --abbrev-ref HEAD")
+    if exists('g:gh_use_canonical')
```

Comment:
Thank you for the PR :) Should we use `if exists('g:gh_use_canonical') && g:gh_use_canonical` ?